pkgs/tools/security/vulnz/default.nix

```
@@ -0,0 +1,115 @@
+{ stdenv
+, fetchFromGitHub
+, gradle
+, jre
+, lib
+}:
+
+let
+  pname = "vulnz";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jeremylong";
+    repo = "Open-Vulnerability-Project";
+    rev = "v${version}";
+    hash = "sha256-87LFCZXruqzZWAMvhYCSEJ+UiYKlhMQmS2bIWdQXcaY=";
+  };
+  deps = stdenv.mkDerivation {
+    name = "${pname}-deps";
+    inherit src;
+    nativeBuildInputs = [ gradle ];
+    buildPhase = ''
+      export GRADLE_USER_HOME=$(mktemp -d)
+      gradle --no-daemon jar
+    '';
+    # Mavenize dependency paths
+    # e.g. org.codehaus.groovy/groovy/2.4.0/{hash}/groovy-2.4.0.jar -> org/codehaus/groovy/groovy/2.4.0/groovy-2.4.0.jar
+    installPhase = ''
+      find $GRADLE_USER_HOME/caches/modules-2 -type f -regex '.*\.\(jar\|pom\)' \
+        | perl -pe 's#(.*/([^/]+)/([^/]+)/([^/]+)/[0-9a-f]{30,40}/([^/\s]+))$# ($x = $2) =~ tr|\.|/|; "install -Dm444 $1 \$out/$x/$3/$4/$5" #e' \
+        | sh
+    '';
+
+    # Compensate for the `move-docs.sh` setup hook - is there a way to
+    # prevent it from running entirely?
+    postFixup = ''
+      mv $out/share/info $out/info
+    '';
+
+    outputHashMode = "recursive";
+    outputHash = "sha256-/QbfQQCQqSx+P0GKl3fz66Qy3WTV/a8rIBR95U/fXSo=";
+  };
+in stdenv.mkDerivation (finalAttrs: {
+  inherit pname version src;
+
+  nativeBuildInputs = [ gradle ];
+
+  buildPhase = ''
+    depsDir=$(mktemp -d)
+    cp -R ${deps}/* $depsDir
+    chmod -R u+w $depsDir
+
+    gradleInit=$(mktemp)
+    cat >$gradleInit <<EOF
+      gradle.projectsLoaded {
+        rootProject.allprojects {
+          buildscript {
+            repositories {
+              clear()
+              maven { url '$depsDir' }
+            }
+          }
+          repositories {
+            clear()
+            maven { url '$depsDir' }
+          }
+        }
+      }
+
+      settingsEvaluated { settings ->
+        settings.pluginManagement {
+          repositories {
+            maven { url '$depsDir' }
+          }
+        }
+      }
+    EOF
+
+    export GRADLE_USER_HOME=$(mktemp -d)
+    echo using deps from ${deps}
+    gradle --offline --no-daemon --info --init-script $gradleInit vulnz:bootJar
+   '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out $out/share
+    ls vulnz
+    cp vulnz/build/libs/vulnz-${version}.jar $out/share
+
+    mkdir $out/bin
+    echo "${jre}/bin/java -jar $out/share/${pname}-${version}.jar \$*" > $out/bin/vulnz
+    chmod a+x $out/bin/vulnz
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Remove nested `with`s:

```suggestion
  meta = {
```